### PR TITLE
GHCR image push

### PR DESCRIPTION
**Pull request by Praveenraj-K**
-Image will be pushed to DockerHub & GHCR
-updated the code so that it will only trigger when workflow trigger happens in the main branch

### CODE REVIEW

**Comment by devon-sil** [comment]
Thanks.  I've added the developers for review.

File: .github/workflows/build-and-publish.yml [hunk 1]
```diff
@@ -2,36 +2,59 @@ name: Build and Publish
 
 on:
   push:
+    branches:
+      - main
```
Comment (briskt) [inline .github/workflows/build-and-publish.yml]: This should include `tags` as well

File: .github/workflows/build-and-publish.yml [hunk 2]
```diff
@@ -2,36 +2,59 @@ name: Build and Publish
...
+      # Extract metadata (tags, labels) for Docker images
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+          images: |
+            ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}   # For DockerHub
+            ghcr.io/${{ github.repository }}                             # For GHCR
+          tags: |
```
Comment (briskt) [inline .github/workflows/build-and-publish.yml:36]: This should include `semver` or `type=ref,event=tag` as well.

**Review by briskt — CHANGES_REQUESTED** [review]
same comments as other repos

**Review by briskt — APPROVED** [review]

**Pull request merged**